Javatrix.java

```diff
@@ -481,5 +481,27 @@ public Javatrix times(double scalar)
         }
         return actual;
     }
+
+    /**
+     * Matrix transpose.
+     *
+     * @return A'
+     */
+    public Javatrix transpose()
+    {
+        int rows = this.getColumnDimension();
+        int cols = this.getRowDimension();
+        Javatrix actual = new Javatrix(rows, cols);
+        double a;
+        for (int i = 0; i < rows; i++)
+        {
+            for (int j = 0; j < cols; j++)
+            {
+                a = this.get(j, i);
```

Comment:
You are overriding elements. When you get j,i and put it into i,j what happens to the element in i,j, you are not storing it anywhere to put into j,i. And this only works if the matrix is square. If you have a 2x3 matrix and you transpose it, it becomes a 3x2 matrix.